libs/category/src/filters/behaviors/apply/apply-request-to-filters.ts

```diff
@@ -0,0 +1,18 @@
+import { Dict } from '@daffodil/core';
+
+import {
+  DaffCategoryFilterRequest,
+  DaffCategoryFilter,
+} from '../../../models/public_api';
+import { daffApplyFilter } from './apply-filter';
+
+/**
+ * Applies a {@link DaffCategoryFilterRequest} to a {@link Dict} of {@link DaffCategoryFilter}
+ * returning the updated {@link Dict}.
+ */
+export const daffApplyRequestToFilters = (request: DaffCategoryFilterRequest, filters: Dict<DaffCategoryFilter>):  Dict<DaffCategoryFilter> => ({
```

Comment:
I guess @idempotent should be added here and to some of the other functions, since it was added to the remove and toggle functions

Reply:
Added in 5df980eb6e23937e7c8e3583e8afd6228be9f47d